batch7crud-roll8/src/main/webapp/css/style.css

```diff
@@ -0,0 +1,184 @@
+/* Sign In Page*/
+
+.signin-form {
+  width: 40%;
+  margin: 50px auto;
+  background-color: #F39C12;
+}
+
+.signin-form-content {
+  padding: 70px;
+}
+
+.signin-form-content p {
+  padding: 5px;
+  text-align: center;
+}
+
+.signin-form-content p input {
+  width: 97%;
+  border: none;
+  padding: 5px;
+  font-size: 14px;
+}
+
+.signin-form-content p input.signin-submit {
+  width: 100%;
+  font-size: 20px;
+}
+
+/* User Index Page */
+
+.user-boxes {
+  margin: 100px 0;
+}
+
+.user-boxes img {
+  height: 50px;
+}
+
+.user-boxes a {
+  text-decoration: none;
+}
+
+.user-box-left {
+  float: left;
+  width: 66%;
+}
+
+.user-box-l {
+  float: left;
+  width: 49%;
+}
+
+.user-box-r {
+  float: right;
+  width: 49%;
+}
+
+.user-box-right {
+  float: right;
+  width: 33%;
+}
+
+.user-box-content {
+  padding: 50px 20px;
+  border-radius: 10px;
+}
+
+.user-box-content a h2 {
+  float: left;
+  color: #fff;
+  font-size: 30px;
+  font-weight: bold;
+  padding-top: 12px;
+}
+
+.user-box-content a img {
+  float: right;
+}
+
+.blue {
+  background-color: #00C0EF;
+}
+
+.yellow {
+  background-color: #F39C12;
+}
+
+.red {
+  background-color: #DD4B39;
+}
+
+/* Employee Index Page */
+
+.employee-index-menu a {
+  text-decoration: none;
+  color: #fff;
+  float: right;
+  padding: 10px;
+  border-radius: 5px;
+}
+
+.employee-index-menu a h2 {
+  float: left;
+  font-weight: bold;
+  padding-top: 3px;
+  padding-right: 10px;
+}
+
+.employee-index-menu a img {
+  width: 25px;
+  float: right;
+}
+
+.employee-index-list {
+  width: 80%;
+  margin: 0 auto;
+}
+
+.employee-index-list h2 {
+  font-size: 26px;
+  font-weight: bold;
+  line-height: 50px;
+}
+
+.employee-index-table table tr {
+  line-height: 28px;
+}
+
+.employee-index-table table tr td {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Simplify this over-specified selector [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=css%3Aoverspecific-selectors)